Tools/apiCompat/baseline/ApiCompatBaseline.net45.txt

```diff
@@ -1,2 +1,7 @@
+CannotRemoveAttribute : Attribute 'Newtonsoft.Json.JsonObjectAttribute' exists on 'Microsoft.IdentityModel.Tokens.JsonWebKey' in the contract but not the implementation.
```

Comment:
Please add internals visible to M.IM.S2S and M.IM.S2STokens so we can access this in SAL.

Reply:
Added internal visible to lower level SAL and SAL Tests.